pkg/cloudprovider/aws/instancetype.go

```diff
@@ -37,10 +37,16 @@ type InstanceType struct {
 func (i *InstanceType) Name() string {
 	return aws.StringValue(i.InstanceType)
 }
+
 func (i *InstanceType) Zones() []string {
 	return i.ZoneOptions
 }
 
+func (i *InstanceType) CapacityTypes() []string {
+	// values would be "spot" and/or "on-demand": https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_InstanceTypeInfo.html
```

Comment:
This comment is a bit redundant IMO.

Reply:
As you can tell, I'm pretty big on documentation :) But where else is this documented? I feel like you'd have to trace this back if I didn't call it out. If you don't like this, can I put a link to the EC2 API docs up top, at least?

Reply:
>can I put a link to the EC2 API docs up top, at least?
SGTM